openpype/hosts/houdini/plugins/create/create_unreal_staticmesh.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+"""Creator for Unreal Static Meshes."""
+from openpype.hosts.houdini.api import plugin
+from openpype.lib import BoolDef, EnumDef
+
+import hou
+
+
+class CreateUnrealStaticMesh(plugin.HoudiniCreator):
+    """Unreal Static Meshes with collisions. """
+
+    identifier = "io.openpype.creators.houdini.unrealstaticmesh.fbx"
+    label = "Unreal - Static Mesh (FBX)"
+    family = "staticMesh"
+    icon = "fa5s.cubes"
+
+    default_variants = ["Main"]
+
+    def create(self, subset_name, instance_data, pre_create_data):
+
+        instance_data.update({"node_type": "filmboxfbx"})
+
+        instance = super(CreateUnrealStaticMesh, self).create(
+            subset_name,
+            instance_data,
+            pre_create_data)
+
+        # get the created rop node
+        instance_node = hou.node(instance.get("instance_node"))
+
+        # get parms
+        parms = self.get_parms(subset_name, pre_create_data)
+
+        # set parms
+        instance_node.setParms(parms)
+
+        # Lock any parameters in this list
+        to_lock = ["family", "id"]
+        self.lock_parameters(instance_node, to_lock)
+
+    def get_network_categories(self):
+        return [
+            hou.ropNodeTypeCategory(),
+            hou.sopNodeTypeCategory()
+        ]
+
+    def get_pre_create_attr_defs(self):
+        """Add settings for users. """
+
+        attrs = super().get_pre_create_attr_defs()
```

Comment:
I believe `super()` call without arguments only works in Python 3+ and this doesn't work in Python 2 - which might still be relevant for older Houdini versions?